sql/histograms/cruxInp.sql

```diff
@@ -0,0 +1,46 @@
+#standardSQL
+CREATE TEMPORARY FUNCTION spreadBins(bins ARRAY<STRUCT<start INT64, `end` INT64, density FLOAT64>>)
```

Comment:
All INP bins should be 25ms wide, so we can remove this.

Still needed for other metrics that have older data before bin spreading was built-in.

Reply:
Done!